### ISSUE TITLE
Path issues 

### ISSUE THREAD

**Issue by bengrissinger** [iopipe/turtle#19]
Bens-MBP:~ root# iopipe import --name com.example.SomeScript - <<<'input'
cannot enable tty mode on non tty input
Bens-MBP:~ root# eval $(echo "alias iopipe='docker run --rm -i iopipe/iopipe:trunk'" | tee -a ~/.bashrc)
Bens-MBP:~ root# iopipe import --name com.example.SomeScript - <<<'input'
- exec app import --name com.example.SomeScript -
  2016/01/10 03:08:42 open /root/.iopipe/filter_cache/7d3f9b6284c6f36e77b425cac882e8fbbcc97a4727ec20790853076d0f463453: no such file or directory

Bens-MBP:~ root# ls -l /root
ls: /root: No such file or directory

Bens-MBP:~ root# pwd
/var/root
Bens-MBP:~ root# ls -la
total 32
drwxr-x---   9 root  wheel   306 Jan 11 10:09 .
drwxr-xr-x  25 root  wheel   850 Apr 11  2015 ..
-rw-r--r--   1 root  wheel     3 May 26  2015 .CFUserTextEncoding
-rw-r--r--   1 root  wheel   109 Jan 11 10:10 .bashrc
-r--r--r--   1 root  wheel    10 Sep  9  2014 .forward
drwxr-xr-x   3 root  wheel   102 Dec 29 17:50 .oracle_jre_usage
-rw-------   1 root  wheel  1421 Oct  8 11:51 .sh_history
-rw-------   1 root  wheel   584 Jul 27 15:43 .viminfo
drwx------  11 root  wheel   374 Jul 17 06:02 Library

**Comment by ewindisch** [comment]
The code asks the OS for your user's home directory and place the files into <home>/.iopipe/filter_cache/.  It seems like your OS is reporting that your home directory is /root, but you're actually using /var/root as a home directory instead?

Is /var/root listed as your homedir in /etc/passwd? How about:

``` bash
$ getent passwd | grep ^$USER
```

I will also check to make sure the code creates this directory if it does not exist. It might just presume it's there. (Try 'mkdir -p $HOME/.iopipe/filter_cache')

**Comment by ewindisch** [comment]
Two possible bugs here.

1) we do not create .iopipe/filter_cache when needed in the user's home directory.
2) Go's os/user HomeDir might be broken on MacOS and/or your system? TBD

**Comment by ewindisch** [comment]
Had to acquire a Mac... so it seems that the homedir code is working as expected on _my_ system, but can confirm that .iopipe/filter_cache is not being auto-created.

**Comment by bengrissinger** [comment]
so I think the issue is that I am using Piñata (local newest release of
toolbox) on my mac, and its using xhyve.   I will switch over to a vm and
try again

stay tuned

Regards
Ben

**Comment by ewindisch** [comment]
I just looked at your paste again and realized that you're using the Docker image. That explains the different $HOME paths.  You don't have a '-v' to map the home directory back to the host, which I've since added to the README.  Try '-v $HOME:/root' and that should work!

Also, I've uploaded a MacOS native binary under releases.

EDIT: the docker image is self-updating via automatic builds. It should be updated within 30 minutes or so /w the auto-create .iopipe directory fix.
